resources/public/api/conf/1.0/scenarios/retrieveForeignProperty.md

```diff
@@ -0,0 +1,35 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
```

Comment:
Comment relates to file not this specific line.
I think that we need to be only displaying gov test scenarios for unique errors returned from the backend as the majority of these errors can be produced in the frontend when calling the API